server/gitrest/packages/gitrest-base/src/utils/isomorphicgitManager.ts

```diff
@@ -359,20 +365,29 @@ export class IsomorphicGitManagerFactory implements IRepositoryManagerFactory {
             repoPath,
             this.storageDirectoryConfig.baseDir);
 
-        // Create and then cache the repository
         await isomorphicGit.init({
             fs: fileSystemManager,
             gitdir: directoryPath,
             bare: true,
         });
 
         this.repositoryCache.add(repoPath);
+        const lumberjackBaseProperties = helpers.getLumberjackBasePropertiesFromRepoManagerParams(params);
         const repoManager = new IsomorphicGitRepositoryManager(
             fileSystemManager,
             params.repoOwner,
             params.repoName,
-            directoryPath);
-        winston.info(`Created a new repo for owner ${params.repoOwner} reponame: ${params.repoName}`);
+            directoryPath,
+            lumberjackBaseProperties);
+
+        Lumberjack.info(
+            "Created a new repo",
+            {
+                ...lumberjackBaseProperties,
+                repoOwner: params.repoOwner,
+                repoName: params.repoName,
+                directoryPath,
```

Comment:
Since the directory path can change according to the configurations defined in config.json, it might be useful to log directoryPath here in case it can make it easier to detect misconfiguration issues. For instance, we might accidentally change our configs to have `useRepoOwner` equal to `true`, when we didn't intend to. Errors would happen because of that. Logging `directoryPath1 might be useful to detect that easily.

Reply:
Ah, got it, thanks for the explanation! Makes sense to me